docs/features/software-catalog/well-known-annotations.md

```diff
@@ -222,6 +222,20 @@ definition.
 Specifying this annotation will enable GoCD related features in Backstage for
 that entity.
 
+### periskop.io/name
+
+```yaml
+# Example:
+metadata:
+  annotations:
+    periskop.io/name: pump-station
+```
+
+The value of this annotation is the periskop project name for the given entity.
```

Comment:
reading this it sounds like it'd be suitable to name it `periskop.io/project` perhaps?

Reply:
Changed to `periskop.io/service-name`.